apps/chat/src/components/Common/ApplicationDialog.tsx

```diff
@@ -106,6 +106,9 @@ const ApplicationDialogView: React.FC<Props> = ({
 
   const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
   const [isPublishing, setIsPublishing] = useState(false);
+  const [maxInputAttachmentsValue, setMaxInputAttachmentsValue] = useState(
+    selectedApplication?.maxInputAttachments || '',
```

Comment:
also I see that `maxInputAttachments` field has number type. Maybe we should discuss which value we should put by default instead of empty string (0 or 1 for example)